free-2.md

```diff
@@ -7,44 +7,53 @@ good way to meet their needs.
 
 
 ### What's the need?
-_What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help 
-them?_
+The need is for programmers who want to easily create scripts to help them in their
+everyday workflow. 
 
 
 ### Why a language?
-_Why is a DSL appropriate for your user(s)? How does it address the need?_
+I think a language would be good for this because you could easily specify repeatble
+tasks to run. 
 
 
 ### Why you?
-_What excites you about this idea? How did you come up with it?_
-
+I think this idea is exciting because it could help coders have better access to
+coding / creating scripts on the fly. I came up with it because not being able to
+code short scripts conveniently enough is a problem that I face.
 
 ### Domain
-_Describe the project's domain in five words._
+Script Creation for Workflow Automation
 
 
 ### Interface (syntax)
-_How might the user interact with the language? What does programming look 
-like? Why is this the right way to interact with the problem domain?_ 
+I see the interface as basically defining keyboard macros for things that could
+be useful in creating scripts. For example, quickly bring up a writing pad perhaps
+in the background.
+
+Open TextEditor
+Compile in Python
+Directory ~/Desktop/test
 
 
 ### Operation (semantics)
-_What might happen when a program runs? How does a program interact with the
-user? What kinds of errors might occur, and how might they be communicated to
-the user?_
+When a program runs, it would basically do all the work of setting up your coding
+environment so all you have to do is start writing the code. Also ideally to compile
+and run the code would just require a keyboard macro as well.
 
 
 ### Expressiveness
-_What should be easy to do in this language? What should be possible, but
-difficult? What should be impossible or very difficult?_
+It should be easy to write programs to open and set up custom coding environments. It
+might be possible to use the language to open arbitrary applications but it probably
+would not be easy. It would be impossible to do anything besides opening programs and
+setting systems variables. 
 
 
 ### Related work
-_Are there any other DSLs in this domain? If not, describe how you know there
-aren't and conjecture why not. If so, describe them and provide links. How well 
-do they address the need? Are there any particularly admirable qualities of the
-language? Are there parts of the language you think could be improved?_
+I think currently shell scripts would be the way in which someone would go about
+solving this problem. I think a language could improve on this by making it more
+domain specific. Shell scripts are already pretty specific to working with the 
+operating system but another language could be specific to setting up a coding
+environment. 
```

Comment:
Were you thinking of taking a shell language and stripping it down, or putting something on top of the shell to make it easier to use for this purpose, or something else entirely?